### ISSUE TITLE
Content on Attribute required level is vague

### ISSUE THREAD

**Issue by JimDaly** [MicrosoftDocs/powerapps-docs#1581]
FIELDS
--
ApplicationRequired | 2 | The   attribute is required to have a value. Value = 2.
None | 0 | No   requirements are specified. Value = 0.
Recommended | 3 | It   is recommended that the attribute has a value. Value = 3.
SystemRequired | 1 | The   attribute is required to have a value. Value = 1.

Should be:

ApplicationRequired | 2 | Applications   should require the attribute have a value before saving. Value = 2.
-- | -- | --
None | 0 | No   requirements are specified. Value = 0.
Recommended | 3 | Applications   should indicate that the value is recommended. Value = 3.
SystemRequired | 1 | The   Common Data Service Platform requires that this field have a value. Value =   1.

Common Data Service APIs will not throw an error when an attribute with ApplicationRequired applied does not have a value.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: e5456f91-242e-a657-0dfd-b9c97c4231ec
* Version Independent ID: 398f49bd-15b8-e019-594f-475ef277561e
* Content: [Attribute metadata - Power Apps](https://docs.microsoft.com/en-us/powerapps/developer/common-data-service/entity-attribute-metadata#attribute-requirement-level)
* Content Source: [powerapps-docs/developer/common-data-service/entity-attribute-metadata.md](https://github.com/MicrosoftDocs/powerapps-docs/blob/live/powerapps-docs/developer/common-data-service/entity-attribute-metadata.md)
* Service: **powerapps**
* GitHub Login: @mayadumesh
* Microsoft Alias: **jdaly**

**Comment by phecke** [comment]
@JimDaly I have updated the SDK API which is now live. @KumarVivek will take care of the Web API.

I did alter the ApplicationRequired and SystemRequired descriptions. I do not feel that there should be mention of 'saving', which seems to target interactive users. If that term is indeed needed, it should be applied to values 1, 2, and 3.

Also, I removed mentioned of 'Common Data Service' in value 1 to make it more generic. CDS is being renamed as we speak to DataFlex, and I feel that a more generic reference to the data service removes the need to change names (in the API) now and in the future.
